admin/categories.php

```diff
@@ -487,7 +487,7 @@ function init() {
           <?php echo TEXT_EDIT_CATEGORIES_META_TAGS_INTRO; ?> - <strong><?php echo $cInfo->categories_id ?> <?php echo $cInfo->categories_name; ?></strong>
         </div>
         <div class="form-group">
-            <?php echo zen_draw_label(TEXT_EDIT_CATEGORIES_META_TAGS_TITLE, 'metatags_title[' . $languages[$i]['id'] . ']', 'class="col-sm-3 control-label"'); ?>
```

Comment:
Instead of ripping out the complete tag, make it like this:
```<?php echo zen_draw_label(TEXT_EDIT_CATEGORIES_META_TAGS_TITLE, 'metatags_title[]', 'class="col-sm-3 control-label"'); ?>
this way it stays "linked" to the following inputs

Reply:
By the way, good catch on the bug 👍